frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -157,4 +157,38 @@ function normalizeConstraintName(constraint: string): ForeignKeyConstraint {
   }
 }
 
+function convertToPostgresColumnType(
+  type: string,
+  enums: readonly DMMF.DatamodelEnum[],
+): string {
+  const enumNames = enums.map((e) => e.name)
+
+  if (enumNames.includes(type)) {
+    return 'enum'
+  }
+
+  switch (type) {
+    case 'String':
+      return 'text'
+    case 'Boolean':
+      return 'boolean'
+    case 'Int':
+      return 'integer'
+    case 'BigInt':
+      return 'bigint'
+    case 'Float':
+      return 'double precision'
+    case 'DateTime':
+      return 'timestamp'
+    case 'Json':
+      return 'jsonb'
+    case 'Decimal':
+      return 'decimal'
```

Comment:
Although the exact data type is decimal(65,30), we've used the generic 'decimal' for brevity.
https://www.prisma.io/docs/orm/reference/prisma-schema-reference#decimal

Reply:
The policy should have been to display actual values whenever possible, even when parsing schema.rb.
So I thought it was ok to display `decimal(65,30)`.

Reply:
Thank you!That makes sense.  I have updated the code to support the data types as defined in the reference.
https://github.com/liam-hq/liam/pull/572/commits/6d2686608289738edc0bb0f1573384c00c96c334